addons/modern-home-page/home.css

```diff
@@ -0,0 +1,102 @@
+:root {
+    --ste-blue: #4e97fe;
```

Comment:
Is this a reference to ScratchTools? If so, could you provide a link to the original version?

Reply:
Yes, this is my own addon from ScratchTools which has not been published yet. (I can give the PR link)